kt/plugins/godot-gradle-plugin/src/main/kotlin/godot/gradle/projectExt/setupTasks.kt

```diff
@@ -76,28 +73,30 @@ fun Project.setupTasks() {
             )
             // END: graal native image specific tasks
 
+            val copyJarTask = createCopyJarsTask(
+                packageBootstrapJarTask = packageBootstrapJarTask,
+                packageMainJarTask = packageMainJarTask,
+            )
+
             @Suppress("UNUSED_VARIABLE")
             val buildTask = setupBuildTask(
```

Comment:
I know i was the one who wrote the gradle changes; but the variable here is no longer necessary as we now directly configure the jar tasks instead of the too generic `build` task which does way more than we need anyway.

Regarding this; it would probably also make sense to change the task which get's invoked by the editor plugin upon build. to use a task which produces the jar's rather than `build` which not only produces chars but also runs linter's and tests.
So probably something like `assemble` or `shadowJar`. But that can also be done in a separate PR